cmake/winml_unittests.cmake

```diff
@@ -69,6 +69,7 @@ add_winml_test(
   SOURCES ${winml_test_api_src}
   LIBS winml_test_common
 )
+target_compile_definitions(winml_test_api PRIVATE BUILD_GOOGLE_TEST)
```

Comment:
Where is this used?

Reply:
Used in [test.h line 4](https://github.com/microsoft/onnxruntime/blob/6ad3e74f4afa127c3179d5be466284e8771b3ea4/winml/test/common/test.h#L4) to determine the correct test header which defines the WinML Test preprocessor definitions.